content/client-lib-development-guide/features.textile

```diff
@@ -301,12 +302,16 @@ h3(#realtime-connection). Connection
 ** @(RTN11a)@ Explicitly connects to the Ably service if not already connected
 ** @(RTN11b)@ An error will be indicated if the state is @CLOSING@ as the connection must complete the close request before reconnecting
 * @(RTN12)@ @Connection#close@ function:
-** @(RTN12a)@ Sends a @CLOSE@ @ProtocolMessage@ to the server, sets the state to @CLOSING@ and waits for a @CLOSED@ @ProtocolMessage@ to be received
+** @(RTN12f)@ If the connection state is @CONNECTING@, do the operation once the connection state is @CONNECTED@
+** @(RTN12a)@ If the connection state is @CONNECTED@, sends a @CLOSE@ @ProtocolMessage@ to the server, sets the state to @CLOSING@ and waits for a @CLOSED@ @ProtocolMessage@ to be received
 ** @(RTN12b)@ If the @CLOSED@ @ProtocolMessage@ is not received within the "default realtime request timeout":#defaults, the transport will be disconnected and the connection will automatically move to the @CLOSED@ state
 ** @(RTN12c)@ If the transport is abruptly closed following a @CLOSE@ @ProtocolMessage@ being sent, then the connection will automatically move to the @CLOSED@ state
+** @(RTN12d)@ If the connection state is @DISCONNECTED@, aborts the retry process described in "RTN14d":#RTN14d and moves the connection immediately to the @CLOSED@ state
+** @(RTN12e)@ If the connection state is @SUSPENDED@, aborts the retry process described in "RTN14e":#RTN14e and moves the connection immediately to the @CLOSED@ state
 * @(RTN13)@ @Connection#ping@ function:
+** @(RTN13c)@ If the connection state is @CONNECTING@ or @DISCONNECTED@, do the operation once the connection state is @CONNECTED@
```

Comment:
Can we change the wording to `If the connection state is @CONNECTING@ or @DISCONNECTED@, perform the ping operation once the connection state is @CONNECTED@`